src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,36 @@
+.catalog {
+  display: grid;
+  gap: 48px;
+
+  margin: 0 auto;
+
+  @media (max-width: 488px) {
+    max-width: $card-width;
+    grid-template-columns: repeat(1, $card-width);
+  }
+
+  @media (min-width: 488px) {
+    max-width: 448px;
+    grid-template-columns: repeat(2, $card-width);
+  }
+
+  @media (min-width: 768px) {
+    max-width: 696px;
+    grid-template-columns: repeat(3, $card-width);
+  }
+
+  @media (min-width: 1024px) {
+    max-width: 944px;
+    grid-template-columns: repeat(4, $card-width);
```

Comment:
I am not sure if you need add max-width : 448 px,   max-width: 696px,   max-width: 1024px.     Easier only add  .catalog (justify-content: center;).